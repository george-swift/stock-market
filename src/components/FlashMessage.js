import PropTypes from 'prop-types';

const FlashMessage = ({ children }) => (
  <div className="alert alert-danger" role="alert">
    { children }
  </div>
);

FlashMessage.propTypes = {
  children: PropTypes.node.isRequired,
};

export default FlashMessage;
