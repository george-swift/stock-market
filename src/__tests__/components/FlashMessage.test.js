import '@testing-library/jest-dom/extend-expect';
import renderer from 'react-test-renderer';
import FlashMessage from '../../components/FlashMessage';

describe('Snapshot of the FlashMessage component', () => {
  it('should render correctly', () => {
    const tree = renderer.create(
      <FlashMessage>
        Something went wrong! Try again.
      </FlashMessage>,
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
