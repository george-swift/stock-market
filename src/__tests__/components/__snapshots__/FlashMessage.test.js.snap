// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot of the FlashMessage component should render correctly 1`] = `
<div
  className="alert alert-danger"
  role="alert"
>
  Something went wrong! Try again.
</div>
`;
